### PR TITLE
Feature add game notification

### PR DESCRIPTION
**Pull request by thebettermay**
#16 
- added `successNotification` after adding a game;
- added two more buttons: `To opponents` and `To main page`
- changed the color of dropdowns;

### CODE REVIEW

File: src/routes/AddGame.tsx [hunk 1]
```diff
@@ -81,34 +78,60 @@ export default function AddGame(): JSX.Element {
...
-                  className="btn btn-success mt-4"
-                  onClick={_onCreateGameClick}
-                >
-                  Create Game
-                </button>
+                <div className="p-10 flex flex-col">
+                  <button
+                    className="btn btn-success mt-4"
+                    onClick={_onCreateGameClick}
+                    disabled={opponentScore === currentUserScore}
```
Comment (IvaTsu) [inline src/routes/AddGame.tsx:119]:
```suggestion
                    disabled={currentUserScore === opponentScore ||
typeof currentUserScore === "string" ||
typeof opponentScore === "string"}
```

Comment (IvaTsu) [reply src/routes/AddGame.tsx:119]: as otherwise if you enter only 1 score - the button will be activated, because the result will be falsy.

**Review by IvaTsu — CHANGES_REQUESTED** [review]
Awesome 🔥
Left some comments for some improvements.

File: src/App.css [hunk 2]
```diff
@@ -12,3 +12,7 @@
 .read-the-docs {
   color: #888;
 }
+
+.dropdown-content {
```
[the remark this answers is not in the document]
Comment (thebettermay) [reply src/App.css:16]: Yes, I just thought that all the dropdowns in the app (and will have) have the same colour, so why make the css module for that. But I can move it of course

File: src/routes/AddGame.tsx (hunk 1 above)
Comment (thebettermay) [reply src/routes/AddGame.tsx:119]: Yes, makes sense 👍

Comment (thebettermay) [reply src/routes/AddGame.tsx]: +

File: src/App.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (thebettermay) [reply src/App.css]: +

[the remark this answers is not in the document]
Comment (IvaTsu) [reply src/App.css]: Sounds reasonable, but we do not have any of those yet.

**Review by IvaTsu — APPROVED** [review]

**Pull request merged**